public/components/event_analytics/explorer/visualizations/index.tsx

```diff
@@ -93,16 +93,7 @@ export const ExplorerVisualizations = ({
               paddingSize="none"
               className="vis__leftPanel"
             >
-              <div className="explorerFieldSelector">
-                {!isMarkDown && (
-                  <div
-                    className="explorer__vizDataConfig"
-                    data-test-subj="explorer__vizDataConfig-panel"
-                  >
-                    {renderDataConfigContainer()}
-                  </div>
-                )}
-              </div>
+              {!isMarkDown && <>{renderDataConfigContainer()}</>}
```

Comment:
nit. Can push empty fragment `<></>` inside renderDataConfigContainer() return statement. Will make this a bit a cleaner. 
```
{!isMarkDown && renderDataConfigContainer()}

Reply:
ok changed

Reply:
nvm keep getting a lot of errors even when i move it into the return statement of renderDataConfigContainer, will leave it as is